addons/main/XEH_postInit.sqf

```diff
@@ -35,6 +35,20 @@ if (!GVAR(enable)) exitWith {
 
 GVAR(ammoPenCache) = createHashMap;
 
+["CAManBase", "Local", {
+    params ["_unit", "_isLocal"];
+    if (_isLocal || {!alive _unit || {GVAR(numWearablePlates) isNotEqualTo 0 || {isNull (vestContainer _unit)}}}) exitWith {};
```

Comment:
does `GVAR(numWearablePlates) isNotEqualTo 0 ` need to be changed?

right now it would exit if you can equip plates?